crates/bevy_naga_oil/README.md

```diff
@@ -0,0 +1,203 @@
+# Bevy Naga Oil
+
+Bevy Naga Organized Integration Library (`bevy_naga-oil`) is a crate for combining and manipulating shaders.

```

Comment:
Any reason to mix `_` and `-`? It's already bad that Rust crates use both, but within the same name...

Reply:
Oh no that's just a mistake!